src/Cohesity.Powershell/Cohesity.PowerShell.psd1

```diff
@@ -8,7 +8,7 @@
 RootModule = 'Cohesity.PowerShell.dll'
 
 # Version number of this module.
-ModuleVersion = '1.9.8'
+ModuleVersion = '1.9.9'
```

Comment:
Where are the actual changes in your code? I don't see them.

And why has the documentation changed so much ? Look into this.